content/source/docs/enterprise/private/migrate.html.md

```diff
@@ -9,6 +9,24 @@ sidebar_current: "docs-enterprise2-private-installer-migration"
 This document outlines the procedure for migrating from the AMI-based Private Terraform Enterprise (PTFE)
 to the Installer-based PTFE.
 
+## Backup
+
+Before beginning, it's best create an additional backup of your RDS database. This will allow you to rollback the data and continue to use the AMI if necessary.
+
+To do so, go to the [Amazon RDS Instances](https://console.aws.amazon.com/rds/home?region=us-east-1#dbinstances:). You may need to change the region that you are viewing in order to see your PTFE instance. Once you find it, click on the instance name. On the next page, select **Instance Actions** and then **Take snapshot**. On the **Take DB Snapshot** page, enter a name for the snapshot such as _pre installer migration_, and then click **Take Snapshot**.
+
+The snapshot will take a little while to create. After it has finished, you can continue with the rest of the migration steps.
+
+### Reverting back to AMI
+
+If you wish to revert back to the AMI after running the migration script, first if you've already manipulated the state file to move the resources, you'll need to restore the original state file.
+
+With your original terraform state file in place, return to the [Amazon RDS Snapshots](https://console.aws.amazon.com/rds/home?region=us-east-1#db-snapshots:) and find the snapshot you created before migrating. Click on the snapshot and note it's **ARN** value. Open your **.tfvars** file and add `db_snapshot = "arn-value-of-snapshot"`.
+
+Next you'll  run `terraform apply` to make sure everything is setup. This will result in a new RDS instance being built against the snapshot.. If the EC2 instance is still running from the migration process, be sure to issue a `shutdown` on it to get a new instance created.
```

Comment:
everything is setup > everything is set up
snapshot.. > snapshot.

This might just be me not being familiar with the setup, but when you say "If the EC2 instance is still running from the migration process..." I wonder if it's more clear to add a step before this (so would be step 3 in the numbers) saying `If the original AMI EC2 instance is still running, issue a shutdown command on it so that this process can create a new EC2 instance.`